index.d.ts

```diff
@@ -18,6 +18,7 @@ export interface Flag<Type extends FlagType, Default, IsMultiple = false> {
 	readonly default?: Default;
 	readonly isRequired?: boolean | IsRequiredPredicate;
 	readonly isMultiple?: IsMultiple;
+	readonly choices?: any[];
```

Comment:
I made the following change.

### Specification

If type is specified, choices should be an array of this type.
If type is not specified, choices can be any 3 types of an array but can’t contain different types. I considered accepting heterogeneous arrays if types are not specified, but choose homogeneous arrays for the following reasons. Shamefully, the biggest reason is my inability to implement, let me know if this will be the problem.

- It seems to require multiple changes in the type definition around flags.
- I think there are few valid cases to use heterogeneous arrays.
- isMultiple also accepts only homogeneous arrays.

### About Code

**index.d.ts**
I renamed `Default` to `Type` because it will also be used for choices so it needs to be a more general name. And I renamed the original `Type` to `LiteralType`.